File: FitTrackApp/app/src/main/java/com/example/fittrackapp/models/Workout.java
package com.example.fittrackapp.models;

public class Workout {
    public int workoutId;
    public int userId;
    public String workoutDate;
    public String notes;

    public Workout(int userId, String workoutDate, String notes) {
        this.userId = userId;
        this.workoutDate = workoutDate;
        this.notes = notes;
    }
}

File: backend/models/Workout.cs
using System.ComponentModel.DataAnnotations;

namespace FitTrackBackend.Models;

public class Workout
{
    [Key]
    public int WorkoutId { get; set; }

    public int UserId { get; set; }

    public DateTime WorkoutDate { get; set; }

    public string Notes { get; set; } = "";

    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}